accessibility/.pa11yci-mobile.json

```diff
@@ -0,0 +1,23 @@
+{
+  "defaults": {
+    "timeout": 240000,
+    "runners": ["axe"],
+    "ignore": ["color-contrast"],
```

Comment:
Is this intentional? What's the reason for ignoring color contrast?

Reply:
It reports false positive color contrast issues (on ootb uswds components) 

it's been an open issue for a while https://github.com/pa11y/pa11y/issues/633